docs/user/guides/subprojects.rst

```diff
@@ -14,24 +14,30 @@ Adding a subproject
 In the admin dashboard for your project, select :guilabel:`Subprojects` from the menu.
 From this page you can add a subproject by typing in the project slug.
 
+Immediately after adding the subproject, its default version will be visible from the link displayed in the list of subprojects.
+
 .. note::
 
    * On |org_brand|, you need to be maintainer of a subproject in order to choose it from your main project.
    * On |com_brand|, you need to have admin access to the subproject in order to choose it from your main project.
 
-Subproject aliases
-~~~~~~~~~~~~~~~~~~
+Editing a subproject
+--------------------
+
+You can edit a subproject at any time by clicking :guilabel:`|:memo:|` in the list of subprojects.
```

Comment:
<img width="714" alt="Screenshot 2022-12-16 at 10 06 39 AM" src="https://user-images.githubusercontent.com/25510/208161232-e3c236d8-32f4-44dc-96f3-2de05487464b.png">
Also it didn't render properly :)

Reply:
I like the direct use of emojis much more, my edit (GEdit :scream: ) supports it very well.

Since it doesn't render properly, I think we have to consider just using emojis directly.

Also, that button should have more than an icon, it's not accessibility-friendly, which makes it not-so documentation friendly at the same time.

Reply:
Using a real emoji now! Incidentally, this was when I found out that my PyCharm doesn't even display emojis :upside_down_face: